language/phpmailer.lang-pt_br.php

```diff
@@ -24,9 +25,9 @@
 $PHPMAILER_LANG['instantiate']          = 'Não foi possível instanciar a função mail.';
 $PHPMAILER_LANG['invalid_address']      = 'Endereço de e-mail inválido: ';
 $PHPMAILER_LANG['invalid_header']       = 'Nome ou valor de cabeçalho inválido';
-$PHPMAILER_LANG['invalid_hostentry']    = 'hostentry inválido: ';
-$PHPMAILER_LANG['invalid_host']         = 'host inválido: ';
-$PHPMAILER_LANG['mailer_not_supported'] = ' mailer não é suportado.';
+$PHPMAILER_LANG['invalid_hostentry']    = 'Hospedagem inválida: ';
+$PHPMAILER_LANG['invalid_host']         = 'Host inválido: ';
+$PHPMAILER_LANG['mailer_not_supported'] = 'O remetente não é suportado.';
```

Comment:
Please check the context for this translation - the name of the mailer is prepended to this string, so it should start with a space and no capital letter.